src/components/mdx/code-block.tsx

```
@@ -0,0 +1,83 @@
+  // Extract language from className (e.g., "language-typescript" -> "typescript")
+  const language = className?.replace("language-", "") || "plaintext";
+
+  const handleCopy = async () => {
+    // Get the text content from the code element
+    let text = "";
+
+    if (typeof children === "string") {
+      text = children;
+    } else if (typeof children === "object" && children !== null) {
+      // Extract text from nested code element
+      const childrenObj = children as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)